src/index.html

```diff
@@ -11,7 +11,177 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    >
+      <fieldset class="group">
+        <legend>Personal information</legend>
+        <div class="form-field">
+          <label class="form-label" for="surname">Surname:
+            <input
+            type="text"
+            name="surname"
+            id="surname"
+            autocomplete="off"
+            required
+            >
+          </label>
+
+        </div>
+
+        <div class="form-field">
+          <label for="name" class="form-label">Name:
```

Comment:
place the text on the new line:
```suggestion
          <label for="name" class="form-label">
            Name:
```